### PR TITLE
Update smart contracts developer docs (chinese)

### PR DESCRIPTION
**Pull request by zhous**
Modified the discription of the first part.

### CODE REVIEW

**Review by Riely007 — APPROVED** [review]

**Comment by corwintines** [comment]
Thanks @zhous, we normally handle language translations in Crowdin so going to pull translation lead in here.

@lukassim should this be ported over to crowdin?

**Comment by minimalsm** [comment]
I think because of the way our translations work with CrowdIn we would probably need to split this into the two paragraphs (with the readability improvements) it originally was? @lukassim

**Comment by lukassim** [comment]
I don't think there would be any issues with leaving it as is, but splitting it up might improve readability.

File: src/content/translations/zh/developers/docs/smart-contracts/index.md [hunk 1]
```diff
@@ -7,9 +7,7 @@ sidebar: true
 
 ## 什么是智能合约？
 
-智能合约只是一个运行在以太坊链上的一个程序。 它是位于以太坊区块链上一个特定地址的一系列代码（函数）和数据（状态）。
-
-智能合约就是一个[以太坊帐户](/developers/docs/accounts/)。 这意味着他们有余额，他们可以通过网络进行交易。 但是，他们无法被人操控，他们是被部署在网络上作为程序运行着。 个人用户可以通过提交交易执行智能合约的某一个函数来与智能合约进行交互。 智能合约能像常规合同一样定义规则，并通过代码自动强制执行。
+智能合约是一种运行在以太坊链上的程序。 它是位于以太坊区块链上一个特定地址的一系列代码（函数）和数据（状态）。智能合约也是一个[以太坊帐户](/developers/docs/accounts/)，我们称之为合约账户。 这意味着它们有余额，它们可以通过网络进行交易。但它们无法被人操控，它们是被部署在去中心化网络节点上作为程序运行着。个人用户可以通过提交交易执行智能合约的某一个函数来与智能合约进行交互。 智能合约能像常规合同一样定义规则，并通过代码自动强制执行。
```
Comment (minimalsm) [inline src/content/translations/zh/developers/docs/smart-contracts/index.md:10]:
```suggestion
智能合约是一种运行在以太坊链上的程序。 它是位于以太坊区块链上一个特定地址的一系列代码（函数）和数据（状态）。

智能合约也是一个[以太坊帐户](/developers/docs/accounts/)，我们称之为合约账户。 这意味着它们有余额，它们可以通过网络进行交易。但它们无法被人操控，它们是被部署在去中心化网络节点上作为程序运行着。个人用户可以通过提交交易执行智能合约的某一个函数来与智能合约进行交互。 智能合约能像常规合同一样定义规则，并通过代码自动强制执行。
```

**Comment by minimalsm** [comment]
Thanks for spotting and fixing this @zhous ✌️.

Be sure to [join the discord](https://discord.gg/CetY6Y4) if you are interested in contributing further to the project or have any questions for the team. And remember to [claim your POAP](https://ethereum.org/en/contributing/#poap) also 🥳!

**Comment by minimalsm** [comment]
@all-contributors please add @zhous for docs, translation

**Pull request merged**